Utils/Dataflow/069_upload2es/load_data.sh

```diff
@@ -44,8 +44,8 @@ while [ -n "$1" ]; do
         BATCHMODE="e"
       else
         BATCHMODE="$2"
-      shift
-      fi;;
+      fi
+      shift;;
```

Comment:
If it was, for example, `-b -E '\0'`, wouldn\`t this version end up eating '-E' and treating `'\0'` as a positional argument (file name)?

Reply:
@mgolosova 
Oh... It seems I forgot completely about the main idea of the batch-mode argument `-b | --batch`. Of course, it can be empty. I made a "fix" as it can't be. Of course, in your example, `-E` will be eaten. I'll revert this false fix.

Thank you!!! :)